.github/workflows/publish-package.yml

```diff
@@ -8,10 +8,10 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: '14.x'
```

Comment:
Node 14 is EOL in like a month - it may be better to try and get something working with v18, which is the next LTS version.